concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -835,7 +835,7 @@ Hale-1973-1798-833	Food_and_Clothing-30	shirt	n	07a.30	1622	SHIRT
 Hale-1973-1798-834	Food_and_Clothing-31	men's lower garment	n	07a.31		
 Hale-1973-1798-835	Food_and_Clothing-32	cumberbund	n	07a.32		
 Hale-1973-1798-836	Food_and_Clothing-33	turban	n	07a.33	3066	SCARF OR TURBAN
-Hale-1973-1798-837	Food_and_Clothing-34	wrap	n	07a.34		
+Hale-1973-1798-837	Food_and_Clothing-34	wrap	n	07a.34	1113	WRAP
```

Comment:
Would unmap because this is in the "food and clothing" section and presumably means "an article of clothing that may be wrapped around a person" (https://www.merriam-webster.com/dictionary/wrap), while [the defintion of `1113 WRAP` in concepticon](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv#L1114) is: "To surround on all sides."

Reply:
We have glosses like _umhüllen_ and _einhüllen_ which can refer to clothes and are mapped to WRAP. I'd leave it as is.

Reply:
Yeah but _umhüllen_ and _einhüllen_ are verbs. Here it says that "wrap" is a noun

Reply:
Yes, I think this refers to some culture-specific piece of clothing in the region. So I'd leave it unmapped.

Reply:
Ah, I see. Unmapping is correct then.